### DISCUSSION TITLE
Gradient dithering

### DISCUSSION THREAD

**Discussion by mgth** [AvaloniaUI/Avalonia#11321] [Q&A]
Is there anyway to use gradient dithering to reduce "step" effects in gradient ?

**Comment by maxkatz6** [comment]
Try to set `RenderOptions.BitmapInterpolationMode="HighQuality"` on your control where gradient is used.
In 11.0-preview8 there are more options like RenderOptions.EdgeMode, RenderOptions.BitmapBlendingMode and RenderOptions.TextRenderingMode to adjust rendering.

**Reply by mgth** [reply]
Thank you for the response, but it does not seem to change the way gradients are rendered.
